templates/core/index.html

```diff
@@ -6,1074 +6,382 @@
-          }}
+        <p>Reliable, self-healing, remotely accessible and recoverable devices which are always up-to-date with mission-critical over-the-air (OTA) updates.</p>
+      </div>
+    </div>
+    <div class="col-3">
+      <div class="p-heading-icon">
+        <div class="p-heading-icon__header is-stacked">
+          <img class="p-heading-icon__img" src="https://assets.ubuntu.com/v1/aec863e2-Off+the+shelf.svg" alt="">
+          <h5 class="p-heading-icon__title">A strong ecosystem of certified hardware</h5>
         </div>
+        <p>Ubuntu Core has been enabled and is continuously tested on leading IoT and edge devices and hardware.</p>
+        <p><a href="/certified">Read more&nbsp;&rsaquo;</a></p>
       </div>
     </div>
   </div>
 </section>
-
-<!-- start generated content -->
-
-<section class="p-strip--suru-accent is-dark">
-  <div class="row">
-    <div class="col-3 p-card u-hide--small u-vertically-center">
-      <div>
-        {{ image (
-          url="https://assets.ubuntu.com/v1/ca395a32-Embedded+Linux+-+white.svg",
-          alt="",
-          width="526",
-          height="300",
-          hi_def=True,
-          loading="lazy"
-          ) | safe
-        }}
-      </div>
+<section class="p-strip--light">
+  <div class="row u-vertically-center">
+    <div class="col-6">
+      <h2 class="u-sv3">Get to market fast with our strong ecosystem</h2>
+      <p>
+        <a class="p-button" href="/certified">Certified hardware</a>
+        <a class="p-button--positive" href="/core/contat-us">Contact us</a>
+      </p>
     </div>
-    <div class="col-8 col-start-large-5">
-      <div class="p-heading-icon--small">
-        <div class="p-heading-icon__header">
-          <div class=" lazyloaded" data-noscript="">
-            {{ image (
-            url="https://assets.ubuntu.com/v1/4ab8ff35-Whitepaper+-+white.svg",
-            alt="",
-            width="32",
-            height="32",
-            hi_def=True,
-            loading="auto"
-            ) | safe
-            }}
-          </div>
-          <p class="p-heading--4">Whitepaper</p>
+    <div class="col-6">
+      <div class="p-logo-section">
+        <div class="p-logo-section__items">
+          <div class="p-logo-section__item"> <img src="https://assets.ubuntu.com/v1/67f81bfe-intel-new-logo.png" class="p-logo-section__logo" style="max-height:120px; padding:10px" /> </div>
+          <div class="p-logo-section__item"> <img src="https://assets.ubuntu.com/v1/8fdc3ebf-raspberry-pi-logo.png" class="p-logo-section__logo" style="max-height:120px; padding:10px" /> </div>
+          <div class="p-logo-section__item"> <img src="https://assets.ubuntu.com/v1/582bee22-avantech-logo.png" class="p-logo-section__logo" style="max-height:120px; padding:10px" /> </div>
+          <div class="p-logo-section__item"> <img src="https://assets.ubuntu.com/v1/75502e5c-NXP-Logo.svg" class="p-logo-section__logo" style="max-height:120px; padding:10px" /> </div>
+          <script>
```

Comment:
This should either be at the end of the file, or in its own JS file - HTML and JS should be separate as much as possible, it's a lot easier to find and reason about JS when it is.